python/examples/send_command.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python3
+
+import logging
+import os
+import socket
+import sys
+from urllib.parse import urlparse
+
+try:
+    import serial
+except ImportError:
+    serial = None
+
+# Add the Python root directory (fusion-engine-client/python/) to the import search path to enable FusionEngine imports
+# if this application is being run directly out of the repository and is not installed as a pip package.
+root_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.insert(0, root_dir)
+
+from fusion_engine_client.messages import *
+from fusion_engine_client.parsers import FusionEngineDecoder, FusionEngineEncoder
+from fusion_engine_client.utils.argument_parser import ArgumentParser
+from fusion_engine_client.utils.bin_utils import bytes_to_hex
+
+
+if __name__ == "__main__":
+    parser = ArgumentParser(description="""\
+Send a command to a Point One device and wait for a response.
+""")
+
+    parser.add_argument(
+        '-i', '--ignore-response', action='store_true',
+        help="Do not wait for a response from the device.")
+
+    parser.add_argument(
+        '-v', '--verbose', action='count', default=0,
+        help="Print verbose/trace debugging messages.")
+
+    parser.add_argument('device',
+                        help="""\
+The path to the target FusionEngine device:
+- [serial://]<device>[:<baud>] - Send over serial (baud rate defaults to 460800).
+- tcp://<hostname>[:<port>] - Send over TCP (port defaults to 30201).
+""".strip())
+    options = parser.parse_args()
+
+    # Configure output logging.
+    logging.basicConfig(format='%(asctime)s - %(levelname)s - %(name)s:%(lineno)d - %(message)s', stream=sys.stdout)
+    logger = logging.getLogger('point_one.fusion_engine.app')
+
+    if options.verbose == 0:
+        logger.setLevel(logging.INFO)
+    elif options.verbose == 1:
+        logger.setLevel(logging.DEBUG)
+    elif options.verbose == 2:
+        logger.setLevel(logging.DEBUG)
+        logging.getLogger('point_one.fusion_engine.parsers').setLevel(logging.DEBUG)
+    else:
+        logger.setLevel(logging.DEBUG)
+        logging.getLogger('point_one.fusion_engine.parsers').setLevel(logging.TRACE, depth=options.verbose - 2)
+
+    # Specify the message to be sent.
+    message = ResetRequest(reset_mask=ResetRequest.HOT_START)
```

Comment:
I think it might be useful to add a couple other example commands as a comment here to illustrate that this same basic structure is used for config control and whatnot. Eventually, it might be worth pulling some of this into a helper function we can reuse in the Quectel runner.

Reply:
Sure, I'll add a few others commented out below this one.